src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -90,10 +90,10 @@ const getIssuer = (token: string, logger: Logger): string | undefined => {
 	return unverifiedClaims.iss;
 };
 
-export const getTokenType = async (url: string, method: string): Promise<TokenType> => {
-	if (await booleanFlag(BooleanFlags.ENABLE_GENERIC_CONTAINERS)) {
+export const getTokenType = async (url: string, method: string, jiraHost: string): Promise<TokenType> => {
+	if (await booleanFlag(BooleanFlags.ENABLE_GENERIC_CONTAINERS, jiraHost)) {
 		return checkPathValidity(url) && method == "GET"
-		|| await checkGenericContainerActionUrl(`${envVars.APP_URL}${url}`) ? TokenType.normal
+		|| await checkGenericContainerActionUrl(`${envVars.APP_URL}${url}`, jiraHost) ? TokenType.normal
```

Comment:
Hi, actually in `|| await checkGenericContainerActionUrl(`${envVars.APP_URL}${url}`, jiraHost`, since the FF is already checked in the `if (await booleanFlag(BooleanFlags.ENABLE_GENERIC_CONTAINERS, jiraHost))`, why do we need to check the ff again?